src/app/design/frontend/Scandiweb/pwa/src/app/component/ContactPage/ContactPage.component.js

```diff
@@ -0,0 +1,116 @@
+                name: __('Contact Us')
+            },
+            {
+                url: '/',
+                name: __('Home')
+            }
+        ];
+
+        updateBreadcrumbs(breadcrumbs);
+    }
+
+    updateHeaderMod() {
+        const { setHeaderState } = this.props;
+
+        setHeaderState({
+            name: WHITE,
+            title: null
+        });
+    }
+
+    render() {
+        const { ContactInfo: { store_email, store_phone, store_working_hours } } = this.props;
+        const { isLoading } = this.props;
+        return (
+            <div block="ContactPage">
+                <Loader isLoading={ isLoading } />
+                <div block="ContactPage" elem="Title">
+                   { __('GET IN TOUCH WITH VGB') }
+                </div>
+                <div block="ContactPage" elem="Group">
+                    <ContactExpandableContent
+                      heading={ __('CALL CENTER VAGABOND') }
+                      mix={ { block: 'Phone' } }
+                    >
+                        <p block="ContactExpandableContent" elem="SubContent">
+                            { store_phone }
+                        </p>
+                        <p block="ContactExpandableContent" elem="SubContent">
+                            { store_working_hours }
+                        </p>
+                    </ContactExpandableContent>
+                    <ContactExpandableContent
+                      heading={ __('E-MAIL SUPPORT') }
+                      mix={ { block: 'Email' } }
+                    >
+                        <p block="ContactExpandableContent" elem="SubContent">
+                            Send us an e-mail and we will contact you ASAP
+                        </p>
+                        <p block="ContactExpandableContent" elem="MainContent">
+                            { store_email }
+                        </p>
+                    </ContactExpandableContent>
+                </div>
+                <div block="ContactPage" elem="Group">
+                    <ContactExpandableContent
+                      heading={ __('STAY SOCIAL') }
+                      mix={ { block: 'Social' } }
+                    >
+                        <p block="ContactExpandableContent" elem="SubContent">
+                        { __('Find us here:') }
+                        </p>
+                        <CmsBlock identifiers={ ['contact-us-social'] } />
+                    </ContactExpandableContent>
+                    <ContactExpandableContent
+                      heading={ __('FORM APPLICATION') }
+                      mix={ { block: 'Form' } }
+                    >
+                    <p block="ContactExpandableContent" elem="SubContent">
+                        Complete the form and we will contact you ASAP:
+                    </p>
+                    <ContactForm />
+                    </ContactExpandableContent>
+                </div>
+            </div>
+        );
+    }
+}
```

Comment:
Please divide render functions into several smaller functions.

```suggestion
            <div block="ContactPage">
                <Loader isLoading={ isLoading } />
                { this.renderContactTitle() }
                <div block="ContactPage" elem="Group">
                    { this.renderCall() }
                    { this.renderEmailSupport() }
                    .......
                </div>
            </div>
        );
    }
}
```